handler.go

```diff
@@ -45,8 +45,11 @@ func (h *Handler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 	case path == "/", path == "/services":
 		h.serveRedirect(res, req, "/services/")
 
-	case path == "/profile":
-		h.serveProfile(res, req)
```

Comment:
Is removing this path expected?

Reply:
I'm proposing that /profile becomes /tree, and we add /flame as well.

Reply:
"Tree view" being the current graph visualization, even though it's not really a tree. (pprof still calls it a tree)

Reply:
Sounds good :+1: